create_amazon_bucket.md

```diff
@@ -0,0 +1,77 @@
+## Create a new amazon bucket
+
+### Bucket
+Use the naming convention `<product-name>-production`, `<product-name>-staging`
+
+#### Region
+We normally default to use `EU-Stockholm` but in theory this isn't as important, so if another region makes more sense choose a different one.
```

Comment:
Yes if you are building a product inside EU, but I'm not sure if this is supposed to be a 'public guide' because if that's the case then if someone outside of EU reads this then an EU server doesn't make sense

Reply:
And I agree that `eu-north-1` should be in the description, but I think we also need the `EU-Stockholm` in there because that is the only thing shown in the dashboard, we can only see `eu-north-1` in the code 😄

Reply:
It is a public repo with guides for us 🤔 
mhhh... good question 🤷‍♀️

Reply:
That's a fair point, it only says `Stockholm` 🤔 

so:
```suggestion
We normally default to use `eu-north-1` aka. `Stockholm` but this may vary depending on where you and/or your customers are, so if another region makes more sense choose a different one.
```